azurerm/internal/services/containers/data_source_kubernetes_service_version.go

```diff
@@ -40,6 +40,12 @@ func dataSourceArmKubernetesServiceVersions() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"allow_preview": {
```

Comment:
I think this'd make more sense as:

```suggestion
			"include_preview": {
```

WDYT? Could we also [add this to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/d/kubernetes_service_versions.html.markdown#L33) too?

Reply:
Sure. I'll flip the default as well so it maintains the existing behavior by default.